ui/app/components/identity/edit-form.js

```diff
@@ -73,12 +73,11 @@ export default Component.extend({
   ).drop(),
 
   willDestroy() {
-    this._super(...arguments);
     const model = this.model;
-    if (!model) return;
-    if ((model.get('isDirty') && !model.isDestroyed) || !model.isDestroying) {
+    if (model && model.get('isDirty') && !model.isDestroyed && !model.isDestroying) {
       model.rollbackAttributes();
     }
+    this._super(...arguments);
```

Comment:
Always call `this._super(...)` last in `willDestroy` otherwise we get errors in tests